games/2048.cpp/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+github.setup            plibither8 2048.cpp 81c0eebaa2e575c7c7943c01958b88896c9da852
+version                 2019-12-12
+revision                0
+categories              games
+maintainers             {mails.ucas.ac.cn:chenguokai17 @chenguokai} openmaintainer
+
+platforms               darwin
+license                 MIT
+description             Fully featured terminal version of the game "2048" written in C++
+long_description        \
+                        Fully featured terminal version of the game "2048" written in C++ \
+                        which is made to run natively on the GNU/Linux and MacOS platforms.
```

Comment:
No need to have this extra line. Please move the description up a line.

Reply:
Hope the latest commit is what you mean

Reply:
Definitely not what I meant. You need to restore the `long_description`. Just make it on 2 lines instead of 3. The initial slash and newline was not necessary.

Reply:
Restored😅 Anything else?